packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx

```diff
@@ -60,17 +60,9 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLIn
     setPopupWidth(width);
   }, [open]);
 
-  // handle input type-to-select
-  const getSearchString = (inputValue: string): string => {
-    // if there are commas in the value string, take the text after the last comma
-    const searchString = inputValue.split(',').pop();
-
-    return searchString?.trim().toLowerCase() || '';
-  };
-
   // set active option and selection based on typing
   const getOptionFromInput = (inputValue: string): OptionValue | undefined => {
-    const searchString = getSearchString(inputValue);
+    const searchString = inputValue.trim().toLowerCase();
```

Comment:
Can inputValue every be null or undefined at runtime? (I know we have strict mode, but a ?. could be good here)

Reply:
Hm, only if the input slot is not an `<input>` (or any other element that has a `value` prop). I'll update it just in case :)